### PR TITLE
Better log

### PR DESCRIPTION
**Pull request by antoine-de**
we now log all request with the status

and we log the exceptions

### CODE REVIEW

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by antoine-de** [comment]
Jenkins, test this please

File: kirin/api.py [hunk 1]
```diff
@@ -51,9 +53,26 @@
                  '/ire',
                  endpoint='ire')
 
-@app.errorhandler(Exception)
-def error_handler(exception):
+
+def log_exception(sender, exception):
```
Comment (xlqian) [inline kirin/api.py]: `sender` not used

Comment (xlqian) [inline kirin/api.py]: ok...

**Comment by xlqian** [comment]
ok  for me

**Pull request merged**